modules/_labs/slider/react/spec/Slider.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Slider, {SliderProps} from '../index';
+
+describe('ActionBar', () => {
+  const cb = jest.fn();
+  afterEach(() => {
+    cb.mockReset();
+  });
+
+  const defaultSliderProps: SliderProps = {
+    max: 100,
+    min: 1,
+    startValue: 50,
+  };
+  test('render with required props', () => {
+    const wrapper = mount(<Slider {...defaultSliderProps} />);
+    expect(wrapper.props().max).toEqual(100);
+    expect(wrapper.props().min).toEqual(1);
+    expect(wrapper.props().startValue).toEqual(50);
+    wrapper.unmount();
```

Comment:
React Testing Library tests the observable output of a component rather than the implementation details. For instance, if you pass in `min` and `max`, the observable output of the DOM is that there is a `min` and `max` set on the `input[type=range]` element.

```tsx
const { findByRole } = render(<Slider min={1} max={100} value={50} />)

const rangeInput = await findByRole('slider')
expect(rangeInput).toHaveAttribute('min', '1')
expect(rangeInput).toHaveAttribute('max', '100')
expect(rangeInput).toHaveValue(50)
```

If the `toHaveAttribute` fails, it will say something like "Expected element to have attribute: min=1" vs "Expected x to equal 1" which is more descriptive. Same with `toHaveValue`